src/v/cluster/rm_stm.h

```diff
@@ -174,6 +174,10 @@ class rm_stm final : public persisted_stm {
     ss::future<std::vector<rm_stm::tx_range>>
       aborted_transactions(model::offset, model::offset);
 
+    inline model::offset max_collectible_offset() override {
```

Comment:
i think that the inline keyword here is redundant/implied

Reply:
Oops. copy-pasta. :ramen: Thanks!